pyproject.toml

```diff
@@ -81,6 +81,7 @@ test = [
     "loompy>=3.0.5",
     "pytest>=6.0",
     "pytest-cov>=2.10",
+    "pytest-nunit",
```

Comment:
Why are we adding a dependency? Is this doing anything, and if not, can we remove this?